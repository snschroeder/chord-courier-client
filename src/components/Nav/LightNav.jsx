import React from 'react';
import { Link } from 'react-router-dom';
import './light-nav.css';

export default function LightNav() {
  return (
    <nav className="nav-bar">
      <ul className="link-list">
        <Link to="/search" style={{ textDecoration: 'none' }}>
          <li className="nav-link">Progression Randomizer</li>
        </Link>
        <Link to="/full-search" style={{ textDecoration: 'none' }}>
          <li className="nav-link">Chords by Key</li>
        </Link>
        <Link to="/about" style={{ textDecoration: 'none' }}>
          <li className="nav-link">About</li>
        </Link>
      </ul>
    </nav>
  );
}
