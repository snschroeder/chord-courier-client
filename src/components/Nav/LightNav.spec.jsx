/* eslint-disable no-undef */
import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { BrowserRouter } from 'react-router-dom';
import LightNav from './LightNav';

describe('LightNav component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(
      <BrowserRouter>
        <LightNav />
      </BrowserRouter>,
      div,
    );
    ReactDOM.unmountComponentAtNode(div);
  });
  it('renders the header correctly', () => {
    const wrapper = shallow(<LightNav />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
