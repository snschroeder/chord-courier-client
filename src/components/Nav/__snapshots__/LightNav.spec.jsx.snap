// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LightNav component renders the header correctly 1`] = `
<nav
  className="nav-bar"
>
  <ul
    className="link-list"
  >
    <Link
      style={
        Object {
          "textDecoration": "none",
        }
      }
      to="/search"
    >
      <li
        className="nav-link"
      >
        Progression Randomizer
      </li>
    </Link>
    <Link
      style={
        Object {
          "textDecoration": "none",
        }
      }
      to="/full-search"
    >
      <li
        className="nav-link"
      >
        Chords by Key
      </li>
    </Link>
    <Link
      style={
        Object {
          "textDecoration": "none",
        }
      }
      to="/about"
    >
      <li
        className="nav-link"
      >
        About
      </li>
    </Link>
  </ul>
</nav>
`;
